SpecialOperations2.Altis/sos/mission/functions/fn_taskEliminateHVT.sqf

```diff
@@ -9,5 +9,90 @@
  *
  */
 
+ private [
+    "_fpb",
+    "_chopperPosition",
+    "_hvtGroup",
+    "_hvt",
+    "_task",
+    "_detected",
+    "_chopper",
+    "_waypoint1",
+    "_trigger",
+    "_waypoint2"
+ ];
+ 
+_fpb = [] call SOS_fnc_getRandomFOB;
 
+_chopperPosition = [
+    SOS_MISSION_AO_POSITION,
+    0.0,
+    SOS_MISSION_AO_RADIUS * 7,
+    2.0,
+    2.0,
+    0,
+    0.0,
+    0.0,
+    SOS_MISSION_BLACKLIST
+] call SOS_fnc_findSafePosition;
+
+_hvtGroup = createGroup east;
+_hvt = _hvtGroup createUnit ["O_officer_F", [0.0, 0.0, 0.0], [], 0, "CAN_COLLIDE"];
+ 
+// Setpos in FPB
+_hvt setPosATL (_fpb buildingPos (floor random count [0, 1, 2, 3]));
+
+// create task
+_task = ["EliminateHVT", position _hvt] call SOS_fnc_addTask;
+
+// Fail Task
+while {alive _hvt} do {
```

Comment:
This is a future bug Cat. Because you call this function, a while loop can only iterate 10000 times. So this means the loop will fail when enough time has gone by. To be specific: 83 minutes ((10000 * 0.5 seconds) / 60).
To prevent this bug from happening place the while loop in the [_hvt, _task] spawn at line 89. When you spawn something, a while loop will run indefinitely.